examples/howtos/howto_ensemble_calculation.ipynb

```diff
@@ -0,0 +1,537 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **SchnetPack Ensemble Calculator**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preparing and loading your data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section prepares the computational environment by importing essential libraries for:\n",
+    "\n",
+    "- **Atomistic simulations** using ASE (Atomic Simulation Environment).\n",
+    "- **Machine learning potentials** via SchNetPack, including ensemble support for uncertainty estimation.\n",
+    "- **MD17 Dataset**, a standard benchmark for ML-based potential energy surfaces.\n",
+    "- **Molecular dynamics (MD)** tools like Langevin dynamics and thermal velocity initialization using Maxwell-Boltzmann distributions.\n",
+    "- **Optimization and analysis utilities**, such as LBFGS optimizer and plotting with `matplotlib`.\n",
+    "\n",
+    "These imports provide the tools to perform uncertainty-aware molecular dynamics simulations with an ensemble of neural network models, enabling deeper insight into the reliability of predicted forces and energies across the MD trajectory.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {},
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "from tqdm.notebook import tqdm\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from ase import units\n",
+    "from ase.io import read\n",
+    "from ase.optimize.lbfgs import LBFGS\n",
+    "from ase.md.velocitydistribution import MaxwellBoltzmannDistribution\n",
+    "from ase.md.langevin import Langevin\n",
+    "\n",
+    "from schnetpack.interfaces.ase_interface import SpkEnsembleCalculator, AbsoluteUncertainty, RelativeUncertainty\n",
+    "import schnetpack.transform as trn\n",
+    "from schnetpack.datasets import MD17"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Predefined datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "SchNetPack supports several benchmark datasets that can be used without preparation.\n",
+    "Each one can be accessed using a corresponding class that inherits from `AtomsDataModule` (a specialized PyTorchLightning `DataModule`), which supports automatic download, conversion and partitioning. Here, we show how to use these data sets at the example of the QM17 benchmark.\n",
```

Comment:
MD17

Reply:
and we don't want to show how to load data. We know that already from the previous tutorials. we want to load the dataset and use it to show case our ensemble calc. and uncertainties